source/coding-the-legislation/35_periods.md

```diff
@@ -1,17 +1,12 @@
-# Periods and instants
+# Periods and Instants
 
-A period can be a month, a year, `n` successive months, `n` successive years or the eternity.
-The smallest unit for OpenFisca periods is the **month**. Therefore:
+A `Period` can be a day, a month, a year, `n` successive days, `n` successive months, `n` successive years or the eternity.  
```

Comment:
With an uppercase letter I infer that you're referring to the Python objects, and I do not think this was the intent here; I would use lowercase here and defer discussion of objects until the next section.